src/test/java/usdot/fhwa/stol/c2c/c2c_mvt/controllers/TestStandardValidationController.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package usdot.fhwa.stol.c2c.c2c_mvt.controllers;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+
+/**
+ *
+ * @author Federal Highway Administration
+ */
+public class TestStandardValidationController 
+{
+	@InjectMocks
+    private StandardValidationController m_oSVC;
+	
+	@BeforeEach
+    public void setUp() 
+	{
+        MockitoAnnotations.openMocks(this);
+		m_oSVC.init();
+    }
+	
+	@Test
+    public void testStandards() 
+	{
+		String sVal = m_oSVC.getStandards();
```

Comment:
Please fix the CI error  related to conversion between `ResponseEntity<String>` and String

Reply:
I removed this test as I was just doing some testing to learn how the framework works. Eric is working on the Unit Tests